data_model/src/transaction.rs

```diff
@@ -1065,30 +1148,33 @@ pub mod error {
         // TODO: Temporarily opaque
         #[ffi_type(opaque)]
         pub enum TransactionRejectionReason {
+            /// Account does not exist
+            AccountDoesNotExist(#[cfg_attr(feature = "std", source)] AccountDoesNotExistError),
```

Comment:
I don't see `#[display]` for this variant, is it done on purpose?

Reply:
It was a mistake, thanks. Btw I wonder why it compiles